datasource_test.go

```diff
@@ -138,3 +138,35 @@ func TestNewElasticsearchDataSource(t *testing.T) {
 		t.Error("datasource creation response should return the created datasource ID")
 	}
 }
+
+func TestNewInfluxDBDataSource(t *testing.T) {
+	server, client := gapiTestTools(t, 200, createdDataSourceJSON)
+	defer server.Close()
+
+	ds := &DataSource{
+		Name:      "foo_influxdb",
+		Type:      "influxdb",
+		URL:       "http://some-url.com",
```

Comment:
Late to the party, but for posterity: every time you need to use a fake domain name, please use one of the [reserved example domain names](https://datatracker.ietf.org/doc/html/rfc2606#section-3): `example.com`, `example.net` and `example.org`. These are protected by RFC2606, whereas something like the one used here can be bought by a malicious actor and you could end up leaking secret information.

Reply:
Good point @inkel I didn't really think about that and just used the same address as the InfluxQL data source test used. 

There seems to be several URLs in used in the tests:

http://grafana.com
http://my-grafana.com
http://some-url.com

I can have a look at cleaning this up at some point if you want to raise another issue for it. It is however only test code so the secrets are just mocked values but there's potentially mileage in Just Doing It for best practice sake :smile: